pkg/storagesvc/storagesvc.go

```diff
@@ -43,6 +49,35 @@ type (
 	}
 )
 
+// Start function
+func Start(logger *zap.Logger, filePath string, port int, storageArgs ...string) error {
+	subdir := os.Getenv("SUBDIR")
+	if len(subdir) == 0 {
+		subdir = "fission-functions"
+	}
+	enableArchivePruner := true
```

Comment:
These should be bubbled up in fission-bundle/main.go before calling storage Service. They can be made configurable in future.

Reply:
So we have two options here. 
- Pass all relevant config params as part of `fission-bundle` cli.
- We just call fission `fission-bundle` with min arguments and rest all config comes from either ENV variables of a config file. Both of these can be done using deployment.yaml in helm chart. 

I'm doing it using second approach. In this way this logic need not be in fission-bundle/main.go . Let me know if this is reasonable ?